packages/dd-trace/src/appsec/iast/vulnerabilities-formatter/utils.js

```diff
@@ -13,15 +13,18 @@ const KEYS_REGEX_WITHOUT_SENSITIVE_RANGES = new RegExp(`"(${STRINGIFY_RANGE_KEY}
 
 const sensitiveValueRegex = new RegExp(DEFAULT_IAST_REDACTION_VALUE_PATTERN, 'gmi')
 
-function iterateObject (target, fn, levelKeys = [], depth = 50) {
+function iterateObject (target, fn, levelKeys = [], depth = 10, visited = new Set()) {
```

Comment:
I reduced it to 10 instead to a 20, because this function is used only when we need to format a nosql vulnerability. The vulnerability is found only in the first levels, so 10 is more than enough to add context for the vulnerability.